plugins/osdn/ovs/plugin.go

```diff
@@ -210,16 +227,26 @@ func (plugin *ovsPlugin) SetUpPod(namespace string, name string, id kubeletTypes
 		return err
 	}
 
-	out, err := utilexec.New().Command(plugin.getExecutable(), setUpCmd, string(id), vnidstr, ingressStr, egressStr, fmt.Sprintf("%t", macvlan)).CombinedOutput()
+	out, err := exec.Command(plugin.getExecutable(), setUpCmd, string(id), vnidstr, ingressStr, egressStr, fmt.Sprintf("%t", macvlan)).CombinedOutput()
 	glog.V(5).Infof("SetUpPod network plugin output: %s, %v", string(out), err)
-	return err
+
+	if isScriptError(err) {
+		return fmt.Errorf("Error running network setup script: %s", getScriptError(out))
```

Comment:
kubernetes adds that when it logs the error:

    message := fmt.Sprintf("Failed to setup network for pod %q using network plugins %q: %v; Skipping pod", format.Pod(pod), dm.networkPlugin.Name(), err)

where `format.Pod(pod)` outputs `fmt.Sprintf("%s_%s(%s)", pod.Name, pod.Namespace, pod.UID)`